report/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "../.tmp/tsbuildinfo/report",
+    "emitDeclarationOnly": true,
+    "declarationMap": true,
+
+    // Limit to base JS and DOM defs.
+    "lib": ["es2020", "dom", "dom.iterable"],
+    // Include `@types/node` for use of `Buffer` in text-encoding.js.
+    "types": ["node"],
```

Comment:
the use of `Buffer` in `text-encoding.js` prevents getting rid of `node` types for these files. I was tempted to write a little type shim just for `Buffer` to keep out `@types/node`, but it seemed a bit too fragile. Maybe worth it at some point

Reply:
> the use of `Buffer` in `text-encoding.js` prevents getting rid of `node` types for these files. I was tempted to write a little type shim just for `Buffer` to keep out `@types/node`, but it seemed a bit too fragile. Maybe worth it at some point

trying out viewer types on top of this, I think we actually do want a little `Buffer` shim. The node types infect everything (we end up with errors on `setTimeout` return types, for instance, if `viewer` doesn't also import `@types/node`, which is just getting silly by that point)